avocado/core/job.py

```diff
@@ -135,10 +129,8 @@ def __init__(self, config=None):
         """
         self.config = config or {}
         self.log = LOG_UI
-        self.loglevel = self.LOG_MAP.get(settings.get_value('job.output',
-                                                            'loglevel',
-                                                            default='debug'),
-                                         logging.DEBUG)
+        self.loglevel = settings.get_value('job.output', 'loglevel',
```

Comment:
I like this change and the idea of removing the custom dict. But, before this change, we had `logging.*` that is an `int` and now we are using just a plain `string`. 

I'm just wondering if this would have effects on other parts of the code.

Reply:
The only occurrence I could find was inside job itself, and it's forwarded to the `logging` library, which gladly accepts that... so I can not think of another side effect of this change.

Reply:
> Hi @clebergnu, this LGTM and it is more than welcome.
> 
> Just one small comment about the logging.

Answered it there.  Also, there is codeclimate warning... which IMO it's worth ignoring (the PR "fixes" 2 issues and "introduces" them again).

Reply:
Ok, if `logging` accepts a string, then, no problem.